### PR TITLE
HELIO-1691 Generalize restricted message (for restricted epubs)

### PR DESCRIPTION
**Pull request by sethaj**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -497,6 +498,14 @@
     t.index ["permission_template_id", "name"], name: "index_sipity_workflows_on_permission_template_and_name", unique: true
   end
 
+  create_table "subscriptions", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```
Comment (conorom) [inline db/schema.rb]: is this related to this work?

Comment (sethaj) [reply db/schema.rb]: Ug. No. Thank you. I'll remove it.
